www/segment-events/index.js

```diff
@@ -0,0 +1,14 @@
+export {
+  ICON_COPIED_EVENT,
+  LEAVE_FEEDBACK_CLICKED_EVENT,
+  PLAYGROUND_EVENTS,
+  CONTRAST_CHECKER_CLICKED_EVENT,
+  PAGE_EDIT_BTN_CLICKED_EVENT,
+  SHADOW_GENERATOR_LAYER_EVENTS,
+  USAGE_INSIGHTS_EVENTS,
+  SETTINGS_EVENTS,
+  EXAMPLE_CODE_BLOCK_WITH_HEADING_EVENTS,
+  EXAMPLE_CODE_BLOCK_WITHOUT_HEADING_EVENTS,
+} from './constants';
```

Comment:
I think you should be able to export everything just with `export * from './constants';`

Reply:
Improved, thanks 👍